src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +163,109 @@ public void testJavadocChild() {
         Assert.assertTrue(javadoc instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.JAVADOC, ((DetailNode) javadoc).getType());
         final Object javadocChild = parseTree.getChild(javadoc, 2);
-        Assert.assertTrue(javadoc instanceof DetailNode);
+        Assert.assertTrue(javadocChild instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.TEXT, ((DetailNode) javadocChild).getType());
     }
 
+    @Test
+    public void testGetIndexOfChild() {
+        final DetailAST nChild = tree.getFirstChild().getNextSibling().getNextSibling();
```

Comment:
I was working off of the UI by running Main.java in the gui package. I understand it doesn't translate clearly here.